test/pages/trust-admin/wards/add.test.js

```diff
@@ -28,7 +28,8 @@ describe("/trust-admin/wards/add", () => {
     });
   });
 
-  describe("AddAWard", () => {
+  /*** hospital drop down will be taken out skip test for now */
+  xdescribe("AddAWard", () => {
```

Comment:
Should this still be skipped?

Reply:
I forgot I skipped that, sorry, it was throwing a fire event warning. But this test will be redundant later because I removed the hospital drop down list on VV-65 branch. I can unskip that for now, though.

Reply:
Updated this test to pass